pkg/manifests/types.go

```diff
@@ -267,6 +267,9 @@ type NodeExporterCollectorConfig struct {
 	// Defines the configuration of the `netdev` collector, which collects network devices statistics.
 	// Enabled by default.
 	NetDev NodeExporterCollectorNetDevConfig `json:"netdev,omitempty"`
+	// Defines the configuration of the `buddyinfo` collector, which collects statistics about memory fragmentation.
```

Comment:
@bburt-rh is it useful to mention that the list of metrics (in fact only one, `node_buddyinfo_blocks`) and that the metrics are collected from `/proc/buddyinfo`?

Reply:
> @bburt-rh is it useful to mention that the list of metrics (in fact only one, `node_buddyinfo_blocks`) and that the metrics are collected from `/proc/buddyinfo`?

@simonpasquier I think it's worth mentioning. I'll add some suggested wording.

Reply:
Document update in the new commit :)